function testFunc() {
    throw 'this is an uncaught error';    
}
